.github/workflows/pr.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       - conda-python-build
       - conda-python-tests
       - docs-build
-      - devcontainer
+#     - devcontainer
```

Comment:
Devcontainer CI will have to remain disabled until cudf is updated with https://github.com/rapidsai/cudf/pull/13815, due to the conflicting `nvcomp` dependency which creates circular conflicts between cudf and kvikio.